src/App.tsx

```diff
@@ -75,13 +75,14 @@ const App = () => {
           width={width}
           height={height}
           numberOfPieces={300}
+          canvasRef={React.createRef()}
         />
       ) : null}
 
       <div className="row column">
         <HomeShowcase />
         <SearchForm
-          handleChange={e => setUserName(e.target.value)}
+          handleChange={ (e: any) => setUserName(e.target.value)}
```

Comment:
The type `any` is strictly avoided. Please remove it and use ` React.FormEvent<HTMLInputElement>` here.

Reply:
Then it should be `React.ChangeEvent<HTMLInputElement>`, because we already know what `Search` component from `input`, receiving. Using your mentioned type highlighting the error.